### PR TITLE
Fix comment bug and spectrum update

### PR DESCRIPTION
**Pull request by gr455**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Fixed bug where comments on sets were not being deleted
Fixed bug where spectrum could not be edited
Fixed corresponding test

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -404,7 +404,7 @@
   get 'capture/index/:id.:format' => 'capture#index' 
 
   post 'comments/spectraset/:id.:format' => 'comments#spectraset' 
-  delete 'comments/delete/:id.:format' => 'comments#delete' 
+  post 'comments/delete/:id.:format' => 'comments#delete'
```
Comment (alaxalves) [inline config/routes.rb:407]: I think this is actually a delete route :thinking:

**Comment by alaxalves** [comment]
What was the issue that you're fixing here?

**Comment by gr455** [comment]
> What was the issue that you're fixing here? 

The comments on sets wouldn't delete and also spectrums' title/notes wouldn't update before

File: config/routes.rb (hunk 1 above)
Comment (gr455) [reply config/routes.rb:407]: Whoops my bad

**Comment by alaxalves** [comment]
> > What was the issue that you're fixing here?
> 
> The comments on sets wouldn't delete and also spectrums' title/notes wouldn't update before

Hmmm cool, I didn't catch this. Nice one, I'll manually test this out and how it goes. Thx a lot man

**Comment by gr455** [comment]
> > > What was the issue that you're fixing here?
> > 
> > The comments on sets wouldn't delete and also spectrums' title/notes wouldn't update before
> 
> Hmmm cool, I didn't catch this. Nice one, I'll manually test this out and how it goes. Thx a lot man

Yep thanks, I'll reset the delete route and force push

**Pull request merged**